src/Meta/MetaFactory.cpp

```diff
@@ -357,7 +357,50 @@ void MetaFactory::createFromMethod(const clang::ObjCMethodDecl& method, MethodMe
         }
     }
 
-    methodMeta.setFlags(MetaFlags::MethodIsInitializer, method.getMethodFamily() == clang::ObjCMethodFamily::OMF_init); // set MethodIsInitializer flag
+    bool isInitializer = (method.getMethodFamily() == clang::ObjCMethodFamily::OMF_init);
```

Comment:
IMO the brackets around this statement (and the others few lines later) are making the code less readable. This is my personal opinion so if you disagree, feel free to disregard it :)